src/main/fc/core.c

```diff
@@ -959,9 +959,18 @@ void processRxModes(timeUs_t currentTimeUs)
     updateActivatedModes();
 
 #ifdef USE_DSHOT
-    /* Enable beep warning when the crash flip mode is active */
-    if (flipOverAfterCrashActive) {
-        beeper(BEEPER_CRASH_FLIP_MODE);
+    /* Enable beep warning when the crashflip mode is active */
+    if (crashFlipModeActive) {
+        beeper(BEEPER_CRASHFLIP_MODE);
```

Comment:
These are archaic aspects of the code that I avoided touching, to keep the PR simple :-)

I think that since crashflip requires motor reversal, and this is only possible with DShot, and so in the code, crashflip mode can never be active without DShot.  In that sense, this line about the Beeper is, in practice, never going to be relevant unless crashflip mode is permitted, and that requires DShot.

However it does seem strange that this check for DShot exists here.  As I read the code, crashflip can only be made active if DShot is enabled.  The only place `crashFlipModeActive = true;` happens is line 535 of core.c, and that is within not only `#ifdef USE_DSHOT` but also `isMotorProtocolDshot()`.

Although I appreciate your comment, and it is true that we can probably remove the `#ifdef USE_DSHOT` tag, I see no harm in leaving it.